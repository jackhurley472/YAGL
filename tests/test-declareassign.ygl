int test(int q) {
int x = 5;
int y;
y = 10;
int z = x * 2 + y;
printInt(z + q);
}

test(1);
test(5);
test(10);

