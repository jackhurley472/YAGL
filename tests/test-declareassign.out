21
25
30
